### ISSUE TITLE
Can't focus password input field on modal - Firefox 61.0.1

### ISSUE THREAD

**Issue by tserieye** [Dogfalo/materialize#6084]
What do I want :
I would like to have a connection form in a materialize modal, therefore I would like to put user input field and password input field to authenticate the user (in this case the button is in the navbar, but it doesn't work anywhere else either).

What happens :
When I open my modal, by clicking on the button handler, I can't have the focus of my password input field (text is working) because it loses its active class quickly.

What I saw :
Inspecting the HTML code in Firefox (it seems to be working in Chrome), we can see that :
 - If we deactivate the display : none from CSS, all tags appear and we can click and keep focus on the password input field, it's working well.
 - When we open the modal with its button handler
    - If we watch the password input field HTML, we can see when we click on the input, the tag gets the active class, but disappears quickly.

So on, in Firefox quantum 61.0.1, we can't put a password input field on a modal, because it loses its mandatory active class.

The code : 
`
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>Test</title>
  
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
  
  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/js/materialize.min.js"></script>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
  <script type="text/javascript">
    $(function(){
      //Initialisation de la modale
      $('.modal').modal();
      $('.modal').css("width","40%");
    });
  </script>
</head>
<body>
  
  <div class="navbar-fixed">
    <nav>
      <div class="nav-wrapper">
        <a href="#!" class="brand-logo">Test</a>
        <ul class="right">
          <li>
            <a class="modal-trigger" href="#loginModal">Connexion</a>
          </li>
        </ul>
      </div>
    </nav>
  </div>
  
  
  <div id="loginModal" class="modal">
    <div class="modal-content">
      <h4>Renseignez vos identifiants</h4>
      <div class="row">
        <div class="offset-s2 col s5">
          <div class="input-field col s12">
            <input id="user" type="text">
            <label for="user">Identifiant</label>
          </div>
          <div class="input-field col s12">
            <input id="password" type="password">
            <label for="password">Mot de passe</label>
          </div>
        </div>
      </div>
      <div class="modal-footer">
        <a href="#!" class="modal-close waves-effect waves-green btn-flat">Se Connecter</a>
      </div>
    </div>
  </div>
  
</body>
</html>
`
